autorole/autorole.py

```diff
@@ -177,7 +177,7 @@ def format_help_for_context(self, ctx: commands.Context) -> str:
         pass
 
     @autorole.command()
-    async def autorole_info(self, ctx: commands.Context) -> None:
+    async def info(self, ctx: commands.Context) -> None:
```

Comment:
Can you just put `@commands.command(name="info")` instead of changing the function name? I prefer seeing more descriptive function names.